### PR TITLE
Add support to use pyfar coordinates

### PR DESCRIPTION
**Pull request by mberz**
Instead of porting spharpy to work on pyfar coordinates natively, support them by converting to spharpy Coordinates internally. This allows for simpler integration of spharpy into a pyfar based workflow while not depending on changes to pyfar suggested in https://github.com/pyfar/pyfar/pull/429.

### CODE REVIEW

File: spharpy/samplings/coordinates.py [hunk 1]
```diff
@@ -264,6 +265,39 @@ def __len__(self):
         """
         return self.n_points
 
+    def to_pyfar(self):
+        """Export to a pyfar Coordinates object.
+
+        Returns
+        -------
+        pyfar.Coordinates
```
Comment (ahms5) [inline spharpy/samplings/coordinates.py:273]:
```suggestion
        :doc:`pf.Coordinates <pyfar:classes/pyfar.coordinates>`
```

File: spharpy/samplings/coordinates.py [hunk 2]
```diff
@@ -264,6 +265,39 @@ def __len__(self):
...
+            convention='right',
+            unit='met')
+
+    @classmethod
+    def from_pyfar(cls, coords):
+        """Create a spharpy Coordinates object from pyfar Coordinates.
+
+        Parameters
+        ----------
+        coords : pyfar.Coordinates
```
Comment (ahms5) [inline spharpy/samplings/coordinates.py:290]:
```suggestion
        coords : :doc:`pf.Coordinates <pyfar:classes/pyfar.coordinates>`
```

File: spharpy/samplings/coordinates.py [hunk 3]
```diff
@@ -392,3 +431,39 @@ def __repr__(self):
         else:
             repr_string = "Sampling with {} points".format(self.n_points)
         return repr_string
+
+    def to_pyfar(self):
+        """Export to a pyfar Coordinates object.
+
+        Returns
+        -------
+        pyfar.Coordinates
```
Comment (ahms5) [inline spharpy/samplings/coordinates.py:440]:
```suggestion
        :doc:`pf.Coordinates <pyfar:classes/pyfar.coordinates>`
```

File: spharpy/samplings/coordinates.py [hunk 4]
```diff
@@ -392,3 +431,39 @@ def __repr__(self):
...
+
+        return pyfar_coords
+
+    @classmethod
+    def from_pyfar(cls, coords):
+        """Create a spharpy SamplingSphere object from pyfar Coordinates.
+
+        Parameters
+        ----------
+        coords : pyfar.Coordinates
```
Comment (ahms5) [inline spharpy/samplings/coordinates.py:456]:
```suggestion
        coords : :doc:`pf.Coordinates <pyfar:classes/pyfar.coordinates>`
```

**Review by ahms5 — APPROVED** [review]
just small things, looks fine

**Review by tluebeck — APPROVED** [review]
Looks fine for me. Really, nothing to add.

**Pull request merged**